src/styl/_components/_preview.styl

```diff
@@ -106,3 +110,158 @@
             left 0
             right 0
             bottom 0
+
+    &-forward // when preview is forward
+        position relative
+
+        .teaser
+            position absolute
+            left 0
+            bottom _rem(30px)
+
+            &-title
+            &-headline
+                color $color-white
+                text-shadow 0 2px 2px $color-base
+
+            &-title
+                position relative
+                z-index 1
+                font-size 1.6em
+                color $color-white
+
+                @media $breakpoints.md
+                    font-size 2.7em
+
+            &-summary
+                display none
+
+        .media
+            .media-wrap
+                &:before
+                    width 4.5em
+                    height 3.5em
+
+                    @media $breakpoints.md
+                        width 6em
+                        height 4.5em
+
+                    left 50%
+                    transform translateX(-50%)
+                    top 0
+                    margin auto
+                    background-size 60% auto
+
+                &:after
+                    content ''
+                    position absolute
+                    left 0
+                    bottom 0
+                    z-index 0
```

Comment:
Yes, centering with absolute :
- top 0 + margin auto = vertical centering
- left 0 + transform translateX(-50%) = horizontal centering

Reply:
I meant the `z-index`

Reply:
Yes, the zindex must be 0, because it is a background gradient.
If it is 1, it passes over the text of the footer.
![image](https://user-images.githubusercontent.com/19707072/83879761-6e62c880-a73e-11ea-84d7-b05d82ea5b3a.png)

So I prefer to define it to avoid the risks (whether it is modified, or badly supported)

Reply:
But it's the default value, it's useless

Reply:
Scratch that, the default value is default, so in order of the declaration. Anyway, still useless